src/android/app/src/main/jni/id_cache.cpp

```diff
@@ -115,6 +117,13 @@ jmethodID GetDiskCacheLoadProgress() {
     return s_disk_cache_load_progress;
 }
 
+jobject GetJavaLoadCallbackStage(VideoCore::LoadCallbackStage stage) {
+    const auto it = s_java_load_callback_stages.find(stage);
+    ASSERT_MSG(it != s_java_load_callback_stages.end(), "Invalid LoadCallbackStage: {}", stage);
+
+    return it->second;
```

Comment:
More modern C++-ish (and arguably more clear) would be
```suggestion
    ASSERT_MSG(s_java_load_callback_stages.count(stage), "Invalid LoadCallbackStage: {}", stage);
    return s_java_load_callback_stages.at(stage);
```

Reply:
Wouldn't this essentially be traversing the map twice ?

Reply:
I'll call the current implementation "good enough", we can followup with a future PR if it's needed.